homeassistant/components/light/avion.py

```diff
@@ -26,12 +26,13 @@
 DEVICE_SCHEMA = vol.Schema({
     vol.Optional(CONF_NAME): cv.string,
```

Comment:
I think that we should provide a default name
```python
DEFAULT_NAME = 'Avi-On'
...
    vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
```

Reply:
The library takes care of this already, so providing our own default name seems redundant.

https://github.com/antsar/python-avion/blob/0.9.1/avion/__init__.py#L88